node_modules/oae-admin/manageuser/bundles/default.properties

```diff
@@ -1,9 +1,13 @@
 ACTIONS = Actions
 ADMINISTRATION = Administration
 ADMINISTRATOR = Administrator
+ARE_YOU_SURE_YOU_WANT_TO_DELETE_USER_NAME = Are you sure you want to delete <strong>${displayName}?</strong>
 BECOME_USER = Become user
 BECOME_USER_NAME = Become ${displayName}
 COULD_NOT_BECOME_USER_NAME = Could not become ${displayName}.
+COULD_NOT_DELETE_USER_NAME = Could not delete ${displayName}.
```

Comment:
Should the become user stuff change to a similar format as well to follow the UI patterns?

`USER_BECAME`, `USER_BECOME_FAIL` etc...?